lib/client-assets.php

```diff
@@ -973,7 +973,8 @@ function gutenberg_register_vendor_scripts() {
 
 	gutenberg_register_vendor_script(
 		'react',
-		'https://unpkg.com/react@16.6.3/umd/react' . $react_suffix . '.js'
+		'https://unpkg.com/react@16.6.3/umd/react' . $react_suffix . '.js',
+		array( 'wp-polyfill' )
```

Comment:
Seems legit, should we do the same for `react-dom`?

Reply:
@youknowriad As far as in can see, `react` is a dependency of `react-dom`, so it will get `wp-polyfill` through `react`.

Reply:
Sounds good, If it's a direct dependency (which the React docs state I think) I do prefer explicitness.

Reply:
We should plan to backport this change into the 5.0 branch.

Reply:
Putting it on my list.